### DISCUSSION TITLE
Moving city events under city route?

### DISCUSSION THREAD

**Discussion by harshtandiya** [fossunited/fossunited#332] [Ideas]
Right now, every event created is under the route `event/<event_permalink`, should we change it for city communities?

The idea is that something like a foss meetup can come under city route itself, like before. Ex: `hyderabad/<event_permalink>`

Additional: Do we want to move back to routes such as `city_name/year/month` for meetups?

**Comment by RiyaMathew-11** [comment]
I like this idea of `cityname/permalink` - this is somewhat similar to the old workflow where we kept   year/month in the old platform. 

- Having cityname would attribute the event to the chapter
- Personally I felt an overhead when we had additional extensions in the route before - it made the link very long. Like CFP would be -> cityname/year/month/cfp 

- Having a unique permalink is good - with this change - the permalink could even be the eventname as well or unique code/format etc.

**Comment by mngshm** [comment]
I'd suggest this better be the one you suggested  `city_name/<event_permalink>` this would initially keep the url shorter. The older format will still make it very long for ex: `/hyderabad/2023/12`
